.gitattributes

```
@@ -1,8 +1,8 @@
 /core/js/dist/*.js binary
 /core/js/dist/*.js.map binary
 
-/apps/accessibility/js/accessibility.js binary
-/apps/accessibility/js/accessibility.js.map binary
+/apps/accessibility/js/*.js binary
+/apps/accessibility/js/*.js.map binary
```

We could probably even do :stuck_out_tongue: 
```suggestion
/apps/*/js/*.js binary
/apps/*/js/*.js.map binary
```

---

But we still have non-compiled js files :fearful:

---

Ah right :cry: